Item.py

```diff
@@ -1,5 +1,8 @@
+import dateutil.parser
```

Comment:
dateutil is missing from your requirements.txt file (or rather py-dateutil?)

Also:
- selenium needs to be there, and I expected python-dotenv in there (to load the .env file automatically)
- your requirements.txt file is appearing as a binary file in git, can you try recreating it?
- It seems to have a lot of other dependencies in there, are they really all needed?